tests/model_hub_tests/models_hub_common/constants.py

```diff
@@ -30,7 +30,7 @@
 hf_hub_cache_dir = tempfile.gettempdir()
 if os.environ.get('USE_SYSTEM_CACHE', 'True') == 'False':
     no_clean_cache_dir = True
-    os.environ['HUGGINGFACE_HUB_CACHE'] = hf_hub_cache_dir
+    hf_hub_cache_dir = os.environ['HUGGINGFACE_HUB_CACHE']
```

Comment:
What is the purpose of this change? As I understand `HUGGINGFACE_HUB_CACHE` may not exist in general

Reply:
It isn't very clear for me either.
My understanding was that if `USE_SYSTEM_CACHE` is `False`, we don't need to use tempdir, but we use the path from `HUGGINGFACE_HUB_CACHE` environment variable.
But we already have `HF_HUB_CACHE` and `hf_cache_dir` for that. Am I not aware of some libraries' internals?